software/teachee-desktop/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-eframe = "0.19.0"
+eframe = "0.20.1"
```

Comment:
Was this version upgrade automatic? or did you do that get some kind of new feature?

Reply:
I needed a function that was private in the old version but public in the current so I updated